paymentsheet/src/main/java/com/stripe/android/customersheet/StripeCustomerAdapter.kt

```diff
@@ -36,6 +36,11 @@ internal class StripeCustomerAdapter @Inject constructor(
     @Volatile
     private var cachedCustomerEphemeralKey: CachedCustomerEphemeralKey? = null
 
+    private val isGooglePayAvailable: Boolean
+        get() = runCatching {
+            CustomerSessionViewModel.component.configuration.googlePayEnabled
+        }.getOrNull() ?: false
```

Comment:
Why do we need this wrapped?

Reply:
I think we should crash if that happens, then we can get reports and fix it.